packages/neuron-ui/src/components/PasswordRequest/index.tsx

```diff
@@ -68,12 +68,9 @@ const PasswordRequest = () => {
   const onChange = useCallback(
     (e: React.SyntheticEvent<HTMLInputElement>) => {
       const { value } = e.target as HTMLInputElement
-      if (/\s/.test(value)) {
-        return
-      }
       dispatch({
         type: AppActions.UpdatePassword,
-        payload: value,
+        payload: value.trim(),
```

Comment:
If we allow space, while disallow them at the beginning or end of the password?

Reply:
Allow password to start and end with whitespaces in https://github.com/nervosnetwork/neuron/pull/1395/commits/f5e511c0164c67d4961a2f6b18d3794084205afb